tests/admin_views/tests.py

```diff
@@ -969,6 +969,14 @@ def test_adminsite_display_site_url(self):
         self.assertEqual(response.context['site_url'], '/my-site-url/')
         self.assertContains(response, '<a href="/my-site-url/">View site</a>')
 
+    def test_date_hierarchy_no_data(self):
+        """
+        date_hierarchy should not fail if there is no data (#31590)
+        """
```

Comment:
This docstring is redundant with the test case name and we don't link to ticket unless we're dealing with a really complex test that requires additional context.

```suggestion
```